doc/classes/Rect3.xml

```diff
@@ -183,7 +183,7 @@
 			<argument index="0" name="with" type="Rect3">
 			</argument>
 			<description>
-				Combine this [code]Rect3[/code] with another, a larger one is returned that contains both.
+				Combines this [code]Rect3[/code] with another, a larger one is returned that contains both.
```

Comment:
Why not just:

Returns a larger Rect3 that contains this Rect3 and [code]with[/code].